tests/models/app.page.ts

```diff
@@ -0,0 +1,29 @@
+import type { Locator, Page } from '@playwright/test';
+
+const BASE_URL = 'https://indogithubers.vercel.app'
```

Comment:
I think it's better if we use Playwright config for the base url. you can enable it on this section, mas @iamyuu 
```typescript
 // playwright.config.ts

  use: {
    /* Base URL to use in actions like `await page.goto('/')`. */
    // baseURL: 'http://127.0.0.1:3000',
  },
```

since the E2E test mostly for the development env, so please to use local env instead of production one (https://indogithubers.vercel.app)

to make the Playwright running the app first, you need to enable it on this section
```typescript
  // playwright.config.ts

  /* Run your local dev server before starting the tests */
  // webServer: {
  //   command: 'npm run start',
  //   url: 'http://127.0.0.1:3000',
  //   reuseExistingServer: !process.env.CI,
  // },
```

Reply:
the reason using production URL because it's also added on CI

I have tried uncomment `webServer` but the test is sucks, can you review at again? maybe I make a mistake